### PR TITLE
Nevest

### PR DESCRIPTION
**Pull request by cristinaboris**

### CODE REVIEW

File: src/components/Home/index.jsx [hunk 1]
```diff
@@ -1,35 +1,163 @@
...
 
 
 const Home = () => {
+
+  const [drawerOpen, setDrawerOpen] = useState(false);
+
+  const toggleDrawer = (open) => (event) => {
+    if (
+      event.type === 'keydown' &&
+      ((event.key === 'Tab' && !event.shiftKey) || event.key === 'Escape')
```
Comment (Nasta7ya) [inline src/components/Home/index.jsx:28]: delete this piece

File: src/components/Home/index.jsx [hunk 2]
```diff
@@ -1,35 +1,163 @@
...
+    ) {
+      return;
+    }
+    setDrawerOpen(open);
+  };
+
+  const menuItems = [{name:'Home', ref:'/'}, {name: 'Last Minute Deals', ref:'/deals'}, {name:'Category', ref:'/category'}, {name:'Contacts', ref:'/contct'}];
```
Comment (Nasta7ya) [inline src/components/Home/index.jsx:35]: add formating

File: src/components/Home/index.jsx [hunk 3]
```diff
@@ -1,35 +1,163 @@
...
+}))
+
+const NavBarLogo = styled('img')(({theme}) => ({
+  cursor: 'pointer',
+  [theme.breakpoints.down('md')]: {
+    display: 'none'
+  }
+}))
+
+const Linking = styled(Link)(({theme}) => ({
```
Comment (Nasta7ya) [inline src/components/Home/index.jsx:84]: separete this part in file

File: src/components/SpecialSection/SpecialSectionTwo.jsx [hunk 4]
```diff
@@ -1,36 +1,73 @@
...
+import CardMedia from '@mui/material/CardMedia';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 
 
 const SpecialSectionTwo = ({item}) => {
+
+  //scroll to top!!!!!!!!!
+  
+ const toTop = () => {
```
Comment (Nasta7ya) [inline src/components/SpecialSection/SpecialSectionTwo.jsx:20]: remove

File: src/components/SpecialSection/SpecialSectionTwo.jsx [hunk 5]
```diff
@@ -1,36 +1,73 @@
...
+      </CardActions>
+    </Card>
          </Link>
   )
 }
 
 
 //<p>{item.rating.rate}</p>
 export default SpecialSectionTwo
+//<Container>
```
Comment (Nasta7ya) [inline src/components/SpecialSection/SpecialSectionTwo.jsx:57]: comment to remove

File: src/redux/reducer.js [hunk 6]
```diff
@@ -42,6 +43,17 @@ switch(action.type){
                 ...state,
                 card: []
             }
+        case types.SORT_BY_PRICE:
+            return{
+            ...state,
+            items: state.items.slice().sort((a,b)=> b.price - a.price)
```
Comment (Nasta7ya) [inline src/redux/reducer.js:49]: decide where it should be

**Review by Nasta7ya — CHANGES_REQUESTED** [review]

**Comment by Nasta7ya** [comment]
+ add logic with filterItem

File: src/components/Home/index.jsx (hunk 1 above)
Comment (cristinaboris) [reply src/components/Home/index.jsx:28]: done

File: src/components/Home/index.jsx (hunk 2 above)
Comment (cristinaboris) [reply src/components/Home/index.jsx:35]: done

File: src/components/SpecialSection/SpecialSectionTwo.jsx (hunk 4 above)
Comment (cristinaboris) [reply src/components/SpecialSection/SpecialSectionTwo.jsx:20]: done

File: src/components/SpecialSection/SpecialSectionTwo.jsx (hunk 5 above)
Comment (cristinaboris) [reply src/components/SpecialSection/SpecialSectionTwo.jsx:57]: done

File: src/redux/reducer.js (hunk 6 above)
Comment (cristinaboris) [reply src/redux/reducer.js:49]: +

**Pull request merged**